src/libraries/System.Private.CoreLib/src/System/IO/FileStatus.Unix.cs

```diff
@@ -532,7 +532,7 @@ private static long UnixTimeSecondsToNanoseconds(DateTimeOffset time, long secon
             return (time.UtcDateTime.Ticks - DateTimeOffset.UnixEpoch.Ticks - seconds * TicksPerSecond) * NanosecondsPerTick;
         }
 
-        private static string GetHandlePath(SafeFileHandle handle)
-            => handle.Path ?? throw new ArgumentException(SR.Arg_HandleHasNoPath);
+        private static string GetErrorPath(SafeFileHandle handle)
+            => handle.Path ?? "<<handle>>";
```

Comment:
This should not be a special string here. I would expect a null path when there is no path. And if that ends up as an error message, the message should be specific to handles.

Reply:
ok, we can probably remove this function all together (https://github.com/dotnet/runtime/pull/60507/files#r868128223).

The compiler will tell when we're trying to pass a `string?` to where it expects a `string`.

Reply:
@tmds So we don't need this method?

Reply:
See my last commit